RELEASE.md

```diff
@@ -15,28 +15,28 @@ RELEASE-TEMPLATE-BEGIN
 
 In parallel, the following releases can be done:
 
-* che-theia,
-* che-machine-exec,
-* che-devfile registry, then che-plugin-registry (once che-theia and machine-exec are done)
-* che-parent, then che-docs, then che as Release Candidate for QE
+* che-theia
+* che-machine-exec
+    * che-plugin-registry (once che-theia and machine-exec are done)
+* che-devfile registry
+* che-parent, then che-docs, then che (including [quay images](https://github.com/eclipse/che/blob/master/.ci/cico_common.sh#L123-L131)) to release-candidate branch
 
 Then in series:
 
-* che-parent, then che-docs, then che as Release
+* che-parent, then che-docs, then che (copy from release-candidate to release branch)
 * che-operator, then chectl
 
 - [ ] che-theia
-  - [ ] _specific version of theia used: *(tag or sha)*_
 - [ ] che-machine-exec
-- [ ] che-plugin-registry _(depends on che-theia, che-machine-exec)_
+    - [ ] che-plugin-registry _(depends on che-theia, che-machine-exec)_
 - [ ] che-devfile-registry
 - [ ] che-parent
-- [ ] che-docs _(depends on parent)_
-- [ ] che _(depends on docs and parent)_
-- [ ] che-operator _(depends on all of the above)_
+    - [ ] che-docs _(depends on parent)_
```

Comment:
new items "che-dashboard" and "che-workspace-loader" should be added alongside "che-docs", as they're now need before the next step

Reply:
OK, please edit my PR then.